src/VersionChecker.ts

```diff
@@ -157,7 +194,7 @@ export default class VersionChecker {
 
     private async onUpdate() {
         await this.updateDataDragonVersion();
-        await this.updateGameVersion();
+        await this.updateGameVersion2020();
```

Comment:
why

Reply:
cus you might want to swap back some time :shrug:

Reply:
We have version control for those cases